google/cloud/storage/internal/xml_escape.h

```diff
@@ -0,0 +1,37 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_STORAGE_INTERNAL_XML_ESCAPE_H
+#define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_STORAGE_INTERNAL_XML_ESCAPE_H
+
+#include "google/cloud/storage/version.h"
+#include "absl/strings/string_view.h"
+
+namespace google {
+namespace cloud {
+namespace storage_internal {
+GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
+
+/// Escapes a string for using it as a tag name in XML.
+std::string EscapeXmlTag(absl::string_view const& val);
+
+/// Escapes a string for using it as a text in XML.
+std::string EscapeXmlContent(absl::string_view const& val);
```

Comment:
`absl::string_view`s should be passed by value.

I also might call the parameters `tag` and `content` respectively.

Reply:
Done